whichone/__init__.py

```diff
@@ -25,6 +25,12 @@
     elif ('DEFAULT_RATE_LIMITS' not in app.config):
         print("DEFAULT_RATE_LIMITS not found in /config/defaults.py")
         exit(0)
+    elif (app.config['SPOTIFY_CLIENT_ID'] == "YOUR SPOTIFY DEV CLIENT ID HERE"):
+        print("Insert a valid SPOTIFY_CLIENT_ID in /config/defaults.py")
+        exit(0)
+    elif (app.config['SPOTIFY_CLIENT_SECRET'] == "YOUR SPOTIFY DEV CLIENT SECRET'"):
```

Comment:
Can these be passed as command line args?

Reply:
Currently they're only being read trough a config file but that's a good idea. Would you say remove them from the config entirely and just pass them in as command line arguments every time?